server/lib/404.js

```diff
@@ -6,6 +6,7 @@
 
 // It's a 404 not found response.
 
+/*jshint unused: false */
 module.exports = function (req, res, next) {
```

Comment:
I think we should just tweak the `unused` option in .jshintrc (in /.jshintrc, /app/.jshintrc, and now in /server/.jshintrc)
http://jshint.com/docs/options/#unused

I think if we change it from `unused: true` to `unused: vars`, it will ignore unused function arguments and only complain about unused variables (which is probably what we want).